.github/workflows/update-versions.yaml

```diff
@@ -0,0 +1,53 @@
+name: Create PRs for new versions of parts
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 10 * * *"
+
+jobs:
+  update-versions:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        part: ["cni", "containerd", "helm", "runc"]
+
+    steps:
+      - name: Checking out repo
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Fetch and filter upstream tags
+        id: fetch-upstream
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          sudo apt-get -qq update
+          sudo apt-get -qq install -y jq
+
+          repository="$(cat "build-scripts/components/${{ matrix.part }}/repository")"
+          current_tag="$("build-scripts/components/${{ matrix.part }}/version.sh")"
+
+          IFS='.' read -r -a current_tag_parts <<< "$current_tag"
+
+          git clone $repository part
```

Comment:
is `--depth 1` faster here?

Reply:
Wouldn't that not fetch the tags(shallow clone)? We can do `git clone --depth 1` and `git fetch --tags` but not sure if there are any time savings there.